### DISCUSSION TITLE
关于 Akismet 的使用方式？

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#633] [Q&A] [answered]
请问，注册 Akismet 账户时，需要填写一个 URL，应该填写自己的网站的URL，还是 **xxxxx.vercel.app**  vercel的这个URL？
两者在使用上有区别 或 限制吗？

如图：
<img width="472" alt="Screenshot 2021-10-31 at 17 04 49" src="https://user-images.githubusercontent.com/29486126/139575568-9a9df016-1bdd-4d04-9a0d-0774196bc8bf.png">

**Comment by lizheming** [comment, marked as answer]
我理解应该都可以，没有影响

**Reply by ghost** [reply]
我尝试过，使用两种URL注册后，得到的KEY都是一样的。
